extensions/adobe/experience/decisioning/personalized-content-option.schema.json

```diff
@@ -26,7 +26,14 @@
   "description": "Personalized decision options form the set of choices for any decision actvity. The objective for decisioning is to take a large inventory of items and apply numerous constraint rules to that inventory to narrow it down and then to rank the qualifying options according to a criteria. The resulting propositions assemble and personalize the experience for specific individuals.",
   "definitions": {
     "personalized-content-option": {
-      "properties": {}
+      "properties": {
+        "xdm:fCapConstraintsLastIndex": {
+          "type": "integer",
```

Comment:
Same comment as above on the min/max range

Reply:
@anandphatak Updated with min/max values